### ISSUE TITLE
Might I benefit from scale_x_y = 1.05 in tiny_yolo or tiny_yolo_pan2 ?

### ISSUE THREAD

**Issue by LukeAI** [AlexeyAB/darknet#3691]
Might I benefit from putting scale_x_y = 1.05 in to the yolo layers of tiny_yolo or tiny_yolo_pan2 ?

**Comment by AlexeyAB** [comment]
Yes, add scale_x_y = 1.05 scale_x_y = 1.1 scale_x_y = 1.2 in different [yolo] layers in your cfg-file

**Comment by AlexeyAB** [comment]
just to know, in my cfg-files [yolo] layers go in reversed order than in the original cfg-files

**Comment by LukeAI** [comment]
great, I'll try it out! - If this is a "free" accuracy boost why not insert it into every .cfg ?

**Comment by AlexeyAB** [comment]
Because this is new feature.
There are several new "free" accuracy boosts, which are not tested well yet: https://github.com/AlexeyAB/darknet/projects/1

So it will be added in new cfg-files.
